pkg/operator/controllers/checker/checker_controller.go

```diff
@@ -38,7 +38,7 @@ func NewReconciler(log *logrus.Entry, maocli maoclient.Interface, arocli aroclie
 
 	if role == operator.RoleMaster {
 		checkers = append(checkers,
-			NewMachineChecker(log, maocli, arocli, role, isLocalDevelopmentMode),
+			// NewMachineChecker(log, maocli, arocli, role, isLocalDevelopmentMode),
```

Comment:
```suggestion
		
```
This doesn't need to be here anymore because you moved this out of the checker_controller. You can delete this.

Reply:
Fixed, resolving.